src/modules/workspaces/repositories/team.repository.spec.ts

```diff
@@ -129,25 +134,63 @@ describe('SequelizeWorkspaceTeamRepository', () => {
     });
   });
 
-  describe('getTeamsInWorkspace', () => {
-    const workspace = newWorkspace();
-    it('When a workspace id is not found then we receive an empty array', async () => {
-      jest.spyOn(workspaceTeamModel, 'findAll').mockResolvedValueOnce([]);
-      const response = await repository.getTeamsInWorkspace(workspace.id);
-      expect(response).toEqual([]);
+  describe('getTeamAndMemberByWorkspaceAndMemberId', () => {
+    it('When members are not found, then return empty array', async () => {
+      jest.spyOn(workspaceTeamUserModel, 'findAll').mockResolvedValueOnce([]);
+
+      const result = await repository.getTeamAndMemberByWorkspaceAndMemberId(
+        'workspaceiD',
```

Comment:
Try to use data with the most similar - if not identic - format. If the workspace id is a uuid, use a uuid